docs/csharp/tour-of-csharp/snippets/shared/HelloWorld.cs

```diff
@@ -1,5 +1,4 @@
 ﻿using System;
```

Comment:
This should have been a namespace declaration, rather than a `using` statement:

```suggestion
﻿using System;
﻿namespace TourOfCsharp;
```